staging/src/k8s.io/kubectl/pkg/cmd/get/sorter_test.go

```diff
@@ -47,6 +48,61 @@ func encodeOrDie(obj runtime.Object) []byte {
 	}
 	return data
 }
+func createResource(str string) (resource.Quantity, error) {
+	q, err := resource.ParseQuantity(str)
```

Comment:
This could just `return resource.ParseQuantity(str)`... so let's get rid of this function and inline the `resource.ParseQuantity(str)` calls.